pkg/apiserver/proxy.go

```diff
@@ -151,6 +151,8 @@ func (r *ProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 	httpCode = http.StatusOK
 	newReq.Header = req.Header
+	newReq.ContentLength = req.ContentLength
+	newReq.TransferEncoding = req.TransferEncoding
```

Comment:
But we need to keep the new request being "chunked" when it hits wire, right? If so, I think we need to copy the Request.TransferEncoding

The transfer writer will set the transfer-encoding header field to "chunked" if Request.TransferEncoding contains "chunked:
https://golang.org/src/net/http/transfer.go#L177
Same for the Body, the body will be chunked if Request.TransferEncoding contains "chunked":
https://golang.org/src/net/http/transfer.go#L213